model/model.py

```diff
@@ -0,0 +1,63 @@
+"""
+VidSal model
+"""
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision as tv
+from torch.autograd import Variable
+import math
+from model.mdn import MDN1D
+from model.cnn3d import CNN3d
+
+class VidSal(nn.Module):
+    def __init__(self, model_path, hidden_size=256, LSTM_layers=1, freeze_cnn3d_weights =[True,True,True,True,True]):
+        super(CNN3d, self).__init__()
```

Comment:
"super(VidSal, self).__init__()" instead of "super(CNN3d, self).__init__()"

Reply:
Right!